en/docs/reference/samples/api-development-sample/development-of-developer-optimized-apis-sample.md

```diff
@@ -36,30 +34,29 @@ In the future, ABC bank will need more and more functionality exposed through AP
 
-##### Same API accessed by different clients
+### Same API accessed by different clients
 
 In WSO2 API Manager, clients can create separate applications for the same API and in turn use these different applications to access the API. In this example, different users can get the interest rates for bank loans and fixed deposits by subscribing to the applications, which provide the relevant APIs, that the bank has exposed to the public.
 
 Furthermore, the client may be accessing this information via WSO2 API Manager using a mobile device. Therefore, in order to cater to the latter mentioned requirement ABC bank can develop the client application using the Android SDK, and similarly in order to create a client application using Java, they can use the Java SDK by calling the respective SDK methods.
 
 ![](https://lh4.googleusercontent.com/vg1W-nIII1cMKvfzT7Yiy_xphKI2cgq4KCmCKecFSAr4UQ3KmyldyitsgKaN5500q9vu2K2nAWC4VpeimibHAe9rYfzR7n9hDSUckhtJNVKMXPBzdi-JrOsTxcnSir4Gnky9T9XBogpYomMHwQ)
-##### API developer optimizing the API output by identifying the client
 
-API developers can use the headers sent by the client to detect the type of the client (e.g., if the client is a mobile device or not). For example, consider a client who is accessing ABC bank's API’s and performing online account balance checking. In the latter mentioned scenario, if the API developer needs to allow mobile device clients to only access a selected set of data of their account details, and expose all the data to desktop application clients, this can be achieved when using WSO2 API Manager by having a custom sequence in the API definition. For more information on adding custom sequences, see [Adding Mediation Extensions](https://docs.wso2.com/display/AM260/Adding+Mediation+Extensions) .
+### API developer optimizing the API output by identifying the client
+
+API developers can use the headers sent by the client to detect the type of the client (e.g., if the client is a mobile device or not). For example, consider a client who is accessing ABC bank's API’s and performing online account balance checking. In the latter mentioned scenario, if the API developer needs to allow mobile device clients to only access a selected set of data of their account details, and expose all the data to desktop application clients, this can be achieved when using WSO2 API Manager by having a custom sequence in the API definition. For more information on adding custom sequences, see [Adding Mediation Extensions]({{base_path}}/learn/api-gateway/message-mediation/adding-a-class-mediator) .
```

Comment:
There should not be a space before the fullstop.

.......adding-a-class-mediator) .

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/1762/commits/cb2834840dadcffdece6e356763992c34025e896